modules/installation-guide/partials/con_workspace-exposure-strategies.adoc

```diff
@@ -27,13 +27,16 @@ On a {prod-short} server secured using the Transport Layer Security (TLS) protoc
 ifeval::["{project-context}" == "che"]
 == Single-host strategy
 
-This strategy is available on Kubernetes, but not on OpenShift. When it is used, all workspaces are deployed to sub-paths of the main {prod-short} server domain.
+Single-host strategy have two subtypes with different implementation methods. First subtype is named `native`. This strategy is available and default on Kubernetes, but not on OpenShift, since it uses ingresses for servers exposing. The second subtype named `gateway`, works both on
```

Comment:
"This strategy is available and default on Kubernetes, but not on OpenShift, since it uses ingresses for servers exposing" I'm not sure if it is clear from this sentence, that `native` is available only on Kubernetes. It sounds a bit like it's default on Kubernetes and not on OpenShift, but may be available on both. I'd like the docs team to comment this as I wouldn't probably suggest something better :) @themr0c @boczkowska